dev/airflow-github

```diff
@@ -319,11 +319,15 @@ def compare(target_version, github_token, previous_version=None, show_uncherrypi
 @click.argument("previous_version")
 @click.argument("target_version")
 @click.argument("github-token", envvar="GITHUB_TOKEN")
-@click.option("--disable-progress", is_flag=True, help="Disable the progress bar")
-def changelog(previous_version, target_version, github_token, disable_progress):
+@click.option("--disable-progress-bar", is_flag=True, help="Disable the progress bar")
+def changelog(previous_version, target_version, github_token, disable_progress_bar):
     repo = git.Repo(".", search_parent_directories=True)
     # Get a list of issues/PRs that have been committed on the current branch.
     log = get_commits_between(repo, previous_version, target_version)
+
+    if disable_progress_bar:
+        print(f"Processing {len(log)} commits")
```

Comment:
I'm unsure whether I misunderstood the original design, but I guess we want to disable it because we don't want anything other than changelogs in the console.

Reply:
You are correct, but there is basically no indication that the command is stuck or not. Always a good idea to give an indication for the same

Reply:
Yeah. Progress bar is just nice way of visualising things :)